packages/contracts-bedrock/contracts/test/L2ERC721Bridge.t.sol

```diff
@@ -349,3 +376,30 @@ contract L2ERC721Bridge_Test is Messenger_Initializer {
         );
     }
 }
+
+/// @dev A non-compliant ERC721 token that does not implement the full ERC721 interface.
```

Comment:
this kind of devdoc uses less lines but we have standardized on the other type  `/* ... */`